### ISSUE TITLE
Allow to_pandas to return player_id as column on tracking datasets

### ISSUE THREAD

**Issue by rjtavares** [PySport/kloppy#68]
On tracking datasets, `to_pandas `returns a DataFrame with a row per frame. To make further transformations it may be easier to work with a DataFrame with one row per frame per player.

So `to_pandas` could allow a `player_as_index` option that produces this transformation.

This code worked on a metrica dataset:

```
df = dataset.to_pandas()

def add(a,b): return a + b
players = add(*[[p.player_id for p in t.players] for t in dataset.metadata.teams])+['ball']

x_coords = [p+'_x' for p in players]
y_coords = [p+'_y' for p in players]
ball_cords = ['ball_x', 'ball_y']

data_columns = set(df.columns)-set(x_coords)-set(y_coords)-set(ball_cords)

x = df.set_index(list(data_columns))[x_coords]
y = df.set_index(list(data_columns))[y_coords]
x.columns = players
y.columns = players

df = pd.concat([x.stack(),y.stack()], axis=1)
df.columns = ['x', 'y']
```

If you're ok with it I can test it further and submit a pull request.

**Comment by koenvo** [comment]
That's a great additional. 

Maybe `_record_converter` should always return a list of rows and `to_pandas` does a flatMap.

I'm not sure how the performance difference is between:
1. Do the transformation in kloppy (flatMap)
2. Do the transformation in pandas (like you did above)

Do you think people should also be able to specify which columns are index and which are not?

**Comment by rjtavares** [comment]
All the transformations I'm making are vectorized, I think they should be faster than changing the record converter (which would be a loop inside a loop).

Regarding the index, maybe it's better to reset the index before returning the DataFrame. Then you can set whatever you want as index.

**Comment by koenvo** [comment]
@bdagnino you also made changes to the to_pandas part, right? I would like to change helpers into a module, and create a pandas.py for all to_pandas related code.

**Comment by bdagnino** [comment]
@koenvo yes, to add the qualifiers as columns in the resulting dataframe. May be better to wait for #70 to be merged and then move all together then?
